esrally/telemetry.py

```diff
@@ -238,18 +240,19 @@ def java_opts(self, log_file):
             java_opts.append("-XX:+FlightRecorder")
             java_opts.append("-XX:FlightRecorderOptions=disk=true,maxage=0s,maxsize=0,dumponexit=true,dumponexitpath={}".format(log_file))
             jfr_cmd = "-XX:StartFlightRecording=defaultrecording=true"
-            if recording_template:
-                self.logger.info("jfr: Using recording template [%s].", recording_template)
-                jfr_cmd += ",settings={}".format(recording_template)
-            else:
-                self.logger.info("jfr: Using default recording template.")
         else:
             jfr_cmd += "-XX:StartFlightRecording=maxsize=0,maxage=0s,disk=true,dumponexit=true,filename={}".format(log_file)
-            if recording_template:
-                self.logger.info("jfr: Using recording template [%s].", recording_template)
-                jfr_cmd += ",settings={}".format(recording_template)
-            else:
-                self.logger.info("jfr: Using default recording template.")
+        if delay:
+            self.logger.info("jfr: Using delay [%s].", delay)
+            jfr_cmd += ",delay={}".format(delay)
```

Comment:
```suggestion
            jfr_cmd += f",delay={delay}"
```

Reply:
Had used the string.format as the existing code was using the same. 
Using f string would be a better alternative. Will be updating the same for all.